distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
It may be MIT in newer versions, but if the version used in here is this dual license thats fine.  I think the confusing part is that this almost looks like a reflective link (A ->B, B -> A) so someone parsing the licenses may not be clear on the declaration.

I think tracking the upgrade separately makes sense as well.